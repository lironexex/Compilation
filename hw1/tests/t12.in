// illegal strings 3 - unprintable char ascii value in escape sequence
"\u{7E}" //legal printable char
"\u{7F}" //illegal - DEL char is unprintable
