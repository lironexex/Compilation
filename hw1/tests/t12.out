1 COMMENT 1
2 STRING ~
2 COMMENT 1
Error undefined escape sequence u
